evap/rewards/tests/test_views.py

```diff
@@ -57,16 +57,16 @@ def test_redeem_all_points(self):
         form.set(f"points-{self.event1.pk}", 2)
         form.set(f"points-{self.event2.pk}", 3)
         response = form.submit()
-        self.assertContains(response, "You successfully redeemed your points.")
+        self.assertContains(response, "You successfully redeemed your points.", status_code=200)
```

Comment:
I'd expect that `form.submit()` already expects a 200 status code and fails otherwise, since this is convention with the test framework (however, I have to say, it does not seem to be documented at all, I could only find a reference in the actual code [here](https://github.com/Pylons/webtest/blob/561ef780f695c3a602be7b49486eabde0832373c/webtest/app.py#L675-L681)).